src/Model_Hydro/MHD_SameInterfaceB.cpp

```diff
@@ -0,0 +1,59 @@
+#include "GAMER.h"
+
+#if ( MODEL == HYDRO  &&  defined MHD )
+
+
+
+
+//-------------------------------------------------------------------------------------------------------
+// Function    :  MHD_SameInterfaceB
+// Description :  Ensure that adjacent patches on the same level (i.e., sibling patches) have *exactly*
+//                the same B field on their shared interfaces
+//                --> Even round-off errors are the same
+//
+// Note        :  1. Applied to both real and buffer patches
+//                2. Invoked by Init_GAMER() and Main()
+//                3. Controlled by the runtime option OPT__SAME_INTERFACE_B
+//                4. Always use the B field on the +x/+y/+z sides to overwite that on the -x/-y/-z sides
```

Comment:
overwite -> overw`r`ite. Also in line 41.

Reply:
Fixed